README.md

```diff
@@ -87,6 +87,29 @@ a small, handy program that also uses the
 It will return a YAML list of names and descriptions for all of the
 repositories belonging to an organization.
 
+## `.about.yml` file testing
+
+The `run_about_yml_check` Rake task provides a relatively simple way to check
+a project's `.about.yml` file for errors.
+
+### Add the task to your Rakefile
+
+To test whether the `.about.yml` file for a repository will successfully parse
+based on the schema, simply include the `about_yml` gem in your dependencies and
+then load the task in your Rakefile. If you already include `about_yml`, make
+sure it is using version 0.0.8 or later. See [this commit on 18F/hub](https://github.com/18F/hub/commit/11527e7f9dc63bec9e373ef47860d46c8a991fed)
+for an example implementation.
```

Comment:
Would it make more sense to just include the three lines of code here instead of referencing the commit?  Actually, we probably only need the two for the Rakefile, you already mention using version `0.0.8` or greater in the text here.